docs/specs/validation/content-validation.yml

```diff
@@ -136,3 +136,36 @@ outputs:
   file1.json:
   a/b/c/file.json:
 ---
+# Validate redirected link
```

Comment:
>Validate redirected link [](start = 2, length = 24)

how about moving it to file image-validtion.yml and renaming that file to link-validation.yml?
because they call the same validation interface.

Reply:
good idea! since I don't want to create new link-validation.yml which has only one rule...